code/modules/clothing/chameleon.dm

```diff
@@ -368,6 +372,12 @@
 			var/obj/item/gun/G = gun_type
 			src.gun_choices[initial(G.name)] = gun_type
 
+/obj/item/gun/energy/chameleon/change(picked in gun_choices) //Making the gun change its help text to match the weapon's help text.
```

Comment:
there is a /obj/item/gun/energy/chameleon/verb/change(picked in gun_choices) ahead, you should probably change the name of this